src/dbnode/storage/dirty_series_new_map_gen.go

```diff
@@ -25,60 +25,34 @@
 package storage
 
 import (
-	"github.com/m3db/m3/src/x/ident"
-	"github.com/m3db/m3/src/x/pool"
+	"bytes"
 
 	"github.com/cespare/xxhash/v2"
 )
 
 // dirtySeriesMapOptions provides options used when created the map.
 type dirtySeriesMapOptions struct {
 	InitialSize int
-	KeyCopyPool pool.BytesPool
```

Comment:
nit: As far as I can tell, `dirtySeriesMapOptions` is never actually initialized with values, is this correct?

Reply:
Correct yeah, I'll remove it entirely.